test/tests/functional/pbs_preemption.py

```diff
@@ -202,6 +202,23 @@ def test_preempt_requeue(self):
         """
         self.submit_and_preempt_jobs(preempt_order='R')
 
+    def test_preempt_requeue_exclhost(self):
+        """
+        Test that a job is preempted by requeue on node
+        where attribute share is set to force_exclhost
+        """
+        # set node share attribute to force_exclhost
+        self.server.manager(MGR_CMD_DELETE, NODE, None, "")
+        self.mom.delete_vnode_defs()
+        vnode_prefix = self.mom.shortname
+        a = {'resources_available.ncpus': '1',
+             'sharing': 'force_exclhost'}
+        vnodedef = self.mom.create_vnode_def(vnode_prefix, a, 0)
+        self.assertNotEqual(vnodedef, None)
+        self.mom.insert_vnode_def(vnodedef, 'vnode.def')
```

Comment:
Is there a reason you jump through the hoops of creating a vnodedef file and inserting it instead of using self.server.create_vnodes() ?